cmd/root.go

```diff
@@ -128,7 +128,7 @@ func NewRootCmd() *RootCmd {
 	flags.StringVar(&rootOpts.Output.Probe, "output-probe", rootOpts.Output.Probe, "filepath where to save the resulting eBPF probe")
 	flags.StringVar(&rootOpts.Architecture, "architecture", runtime.GOARCH, "target architecture for the built driver")
 	flags.StringVar(&rootOpts.DriverVersion, "driverversion", rootOpts.DriverVersion, "driver version as a git commit hash or as a git tag")
-	flags.Uint16Var(&rootOpts.KernelVersion, "kernelversion", rootOpts.KernelVersion, "kernel version to build the module for, it's the numeric value after the hash when you execute 'uname -v'")
+	flags.StringVar(&rootOpts.KernelVersion, "kernelversion", rootOpts.KernelVersion, "kernel version to build the module for, it's the numeric value after the hash when you execute 'uname -v'")
```

Comment:
While this is the desired solution, i am not sure it won't break backward compatibility. 
We should double check (CI integration tests will do that for us as soon as you rebase though!)

Reply:
Yep, again, still in draft mode. I haven't looked at the testing yet. I'm not done with making ubuntu work yet.

Reply:
Done and done